model/implicit_constraints.py

```diff
@@ -89,16 +111,31 @@ def add_mapping_shift_to_demand(self, x, y):
                 quicksum(
                     quicksum(
                         x[e, t_marked, v] for t_marked in self.combined_time_periods
-                        if t - v + 1 <= t_marked <= t
+                        if t - v + self.time_step <= t_marked <= t
+                        if (e,t_marked,v) in x
+                        # 
```

Comment:
Should this be deleted @sondrenaerland ?

Reply:
Disse tre linjene kan fjernes ja!